test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs

```diff
@@ -194,28 +194,12 @@ internal static void GetDirectoryPaths (out string rootSourceDirectory, out stri
 #else
 			var configDirectoryName = "Release";
 #endif
-
-#if NET5_0
-			var tfm = "net5.0";
-#elif NET471
-			var tfm = "net471";
-#else
-			var tfm = "";
-#endif
-
-#if ILLINK
-			// Deterministic builds sanitize source paths, so CallerFilePathAttribute gives an incorrect path.
-			// Instead, get the testcase dll based on the working directory of the test runner.
```

Comment:
I think this comment should be preserved, or remove the `thisFile` parameter from the method

Reply:
Fixed